<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeEditorForm</class>
 <widget class="QMainWindow" name="CodeEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gydrium code editor </string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 47, 48);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>713</width>
         <height>457</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextEdit" name="TextPlaceQTextEdit">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;int main()&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;{&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;}&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="FolderImage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_WhatIsGoinOn">
        <property name="text">
         <string>%For_Example - Save%</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>179</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileQMenu">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="OpenFileQAction"/>
    <addaction name="SaveFileQAction"/>
    <addaction name="SaveAsQAction"/>
    <addaction name="separator"/>
    <addaction name="QuitQAction"/>
   </widget>
   <widget class="QMenu" name="CodeQMenu">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="title">
     <string>Открыть код в</string>
    </property>
    <addaction name="OpenInVisualStudioQAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="title">
     <string>Редактировать</string>
    </property>
    <addaction name="ClearQAction"/>
    <addaction name="separator"/>
    <addaction name="CopyQAction"/>
    <addaction name="PasteQAction"/>
    <addaction name="CuteQAction"/>
   </widget>
   <addaction name="FileQMenu"/>
   <addaction name="menu_3"/>
   <addaction name="CodeQMenu"/>
  </widget>
  <action name="OpenFileQAction">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="iconText">
    <string>Открыть</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="SaveFileQAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="SaveAsQAction">
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="QuitQAction">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="OpenInVisualStudioQAction">
   <property name="text">
    <string>&amp;Visual Studio</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="ClearQAction">
   <property name="text">
    <string>Очистить</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="CopyQAction">
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="PasteQAction">
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="CuteQAction">
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionk">
   <property name="text">
    <string>k</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>QuitQAction</sender>
   <signal>triggered()</signal>
   <receiver>CodeEditorForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClearQAction</sender>
   <signal>triggered()</signal>
   <receiver>TextPlaceQTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CopyQAction</sender>
   <signal>triggered()</signal>
   <receiver>TextPlaceQTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PasteQAction</sender>
   <signal>triggered()</signal>
   <receiver>TextPlaceQTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CuteQAction</sender>
   <signal>triggered()</signal>
   <receiver>TextPlaceQTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
